### PR TITLE
Avoid to rewrite the error

### PR DESCRIPTION
**Pull request by allevo**

### CODE REVIEW

File: examples/package-not-found.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+'use strict'
```
Comment (delvedor) [inline examples/package-not-found.js]: I'll put this files inside a `test` folder :)

File: cli.js [hunk 2]
```diff
@@ -28,30 +28,33 @@ function start (opts) {
...
 }
 
 function runFastify (opts) {
   var file = null
   try {
     file = require(path.resolve(process.cwd(), opts._[0]))
   } catch (e) {
-    console.log(`Cannot find the specified file: '${opts._[0]}'`)
-    stop(1)
+    return module.exports.stop(e)
```
Comment (delvedor) [inline cli.js]: Why?

File: examples/package-not-found.js (hunk 1 above)
Comment (allevo) [reply examples/package-not-found.js]: All tests are inside examples (`incorrect-plugin.js` too)

File: cli.js (hunk 2 above)
Comment (allevo) [reply cli.js]: Because I can ovewrite it during tests

Comment (delvedor) [reply cli.js]: I don't like so much as solution, is not elegant. But for the moment could be ok. :)

Comment (allevo) [reply cli.js]: Me too. But I have not sadly found another solution!

**Pull request merged**